index.html

```diff
@@ -27,7 +27,20 @@
                 <a href="#"><img src="images/logomobile.svg" alt="Logo Mobile"></a>
             </div>
             <div class="menumobile">
-                <a href=""><img src="images/ic_menu.svg" alt="Mobile Menu"></a>
+                <nav class="navbar-mobile">
+                    <div class="hamburguer-menu">
+                        <div class="line line-1"></div>
+                        <div class="line line-2"></div>
+                        <div class="line line-3"></div>
+                    </div>
+                    <ul class="hamb-nav-list">
+                        <li class="hamb-nav-item"><a href="#mobile-index" target="_self" class="nav-link">Hello</a></li>
+                        <li class="hamb-nav-item"><a href="#mobileportfolio" target="_self" class="nav-link">Portfolio</a></li>
+                        <li class="hamb-nav-item"><a href="#mobileabout" target="_self" class="nav-link">About</a></li>
+                        <li class="hamb-nav-item"><a href="#mobilecontact" target="_self" class="nav-link">Contact</a></li>
+                    </ul>
```

Comment:
- [ ] When I click the options, the menu doesn't disappear, please, work on this. Remember the menu should disappear (or close).
![image](https://user-images.githubusercontent.com/20563252/127952752-c3feb741-1f8c-451d-934d-c0333d8b3bec.png)

Reply:
Fixed.